### PR TITLE
July 11 homepage changes

### LINKED ISSUE #8199: 11 July Homepage content programming (Recommended Extensions launch)
To set the homepage for the debut of the Recommended Extensions program, I've [documented](https://docs.google.com/document/d/1Mtlh_fzKd7G25XKk9aR77Mt6eJrFk5CMc7cb8V5tQGc/edit?usp=sharing) all the granular content/copy changes that we'll need to implement for the **11 July** push. 

@bobsilverberg @muffinresearch

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #8199

### CODE REVIEW

File: src/amo/pages/Home/index.js [hunk 1]
```diff
@@ -337,8 +268,43 @@ export class HomeBase extends React.Component {
...
+          addonInstallSource={INSTALL_SOURCE_FEATURED}
+          addons={shelves.popularThemes}
+          className="Home-PopularThemes"
+          header={i18n.gettext('Popular themes')}
+          footerText={i18n.gettext('See more popular themes')}
+          footerLink={{
+            pathname: '/search/',
+            query: {
+              addonType: getAddonTypeFilter(ADDON_TYPE_THEME, {
+                _config: this.props._config,
```
Comment (willdurand) [inline src/amo/pages/Home/index.js]: (I filed https://github.com/mozilla/addons-frontend/issues/8256)

File: src/amo/components/GuidesAddonCard/index.js [hunk 2]
```diff
@@ -71,11 +63,8 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
                       linkToAddon
                     />
                   </span>
-                  {staffPick && (
-                    <span className="GuidesAddonCard-content-header-staff-pick">
-                      <Icon name="trophy" />
-                      <span>{i18n.gettext('Staff Pick')}</span>
-                    </span>
+                  {addon && addon.is_recommended && (
```
[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/components/GuidesAddonCard/index.js]: I think we can rename `staffPick` to `showRecommendedBadge` and keep things as is. WDYT?

File: src/amo/pages/Guides/index.js [hunk 3]
```diff
@@ -148,29 +146,31 @@ export const getSections = ({
                in Firefox’s sidebar.`,
           ),
           exploreMore: i18n.gettext(
-            'Explore more %(linkStart)stab management%(linkEnd)s staff picks.',
+            'Explore more %(linkStart)stab%(linkEnd)s extensions.',
           ),
-          exploreUrl: '/collections/4757633/change-up-your-tabs/',
+          exploreUrl: categoryResultsLinkTo({
```
Comment (willdurand) [inline src/amo/pages/Guides/index.js]: I really like this helper function ❤️

File: src/amo/pages/Home/styles.scss [hunk 4]
```diff
@@ -146,27 +146,3 @@
...
-.Icon-Home-SubjectShelf-watching-videos {
-  background-image: url('./img/watching-videos.svg');
-}
-
-.Icon-Home-SubjectShelf-smarter-shopping {
-  background-image: url('./img/smarter-shopping.svg');
-}
-
-.Icon-Home-SubjectShelf-be-more-productive {
-  background-image: url('./img/be-more-productive.svg');
```
Comment (willdurand) [inline src/amo/pages/Home/styles.scss]: Would it make sense to remove the `svg` files too?

File: tests/unit/amo/pages/TestGuides.js [hunk 5]
```diff
@@ -276,6 +276,40 @@ describe(__filename, () => {
       );
     });
 
+    it.each([['does', 'not empty'], ['does not', 'empty']])(
```
Comment (willdurand) [inline tests/unit/amo/pages/TestGuides.js]: heh, that seems a bit convoluted and fragile but 🤷‍♂

File: src/amo/components/GuidesAddonCard/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/GuidesAddonCard/index.js]: I discussed this with Scott in the gdoc, and he agreed with my approach, but you probably didn't see that as it was marked as resolved. I do not think we should ever force a Recommended Badge to appear for a non-recommended add-on. That would mess with the integrity of the recommended system. I think we always need to allow the database to tell us whether an add-on is recommended or not.

IOW: I think the current implementation in the patch is correct.

File: src/amo/pages/Guides/index.js [hunk 6]
```diff
@@ -247,12 +242,10 @@ export const getSections = ({
           exploreMore: i18n.gettext(
             'Explore among thousands of %(linkStart)sphoto, music & video extensions%(linkEnd)s.',
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/pages/Guides/index.js]: Good catch, thanks!

File: src/amo/pages/Home/index.js [hunk 7]
```diff
@@ -298,17 +240,6 @@ export class HomeBase extends React.Component {
...
-          header={extensionsHeader}
-        >
-          <div className="Home-SubjectShelf-text-wrapper">
-            <h2 className="Home-SubjectShelf-subheading">{extensionsHeader}</h2>
-          </div>
-
-          {this.renderCuratedCollections()}
-        </Card>
-
         <LandingAddonsCard
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/pages/Home/index.js]: Correct. We do not want to use `featured`, but rather use `recommended`.

File: src/amo/pages/Home/styles.scss (hunk 4 above)
Comment (bobsilverberg) [reply src/amo/pages/Home/styles.scss]: Good idea. 👍

File: tests/unit/amo/pages/TestGuides.js (hunk 5 above)
Comment (bobsilverberg) [reply tests/unit/amo/pages/TestGuides.js]: Yeah, this line is a bit confusing, but the output when the tests run is entirely understandable and it works well.

**Comment by bobsilverberg** [comment]
Thanks for the review @willdurand. This is ready for another look.

File: src/amo/components/GuidesAddonCard/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/components/GuidesAddonCard/index.js]: > but you probably didn't see that as it was marked as resolved.

yeah, that's why there is a description attached to a PR usually 😋 

Ok then.

**Pull request merged**